package.json

```diff
@@ -17,6 +17,9 @@
     "submit:android": "eas submit --platform android",
     "submit:ios": "eas submit --platform ios",
     "lint": "./node_modules/.bin/eslint __tests__ src --ext .js,.jsx,.ts,.tsx",
+    "clear-state": "maestro test __maestro__/clear-state.yml -e APP_ID=de.dev.smartvillage.app",
+    "maestro": "maestro test __maestro__ -e APP_ID=de.dev.smartvillage.app",
+    "maestro:clear": "maestro test __maestro__ -e APP_ID=de.dev.smartvillage.app -e CLEAR_STATE=true",
```

Comment:
for the future i plan to call these with passing the `APP_ID` per script as well to be able to be dynamic on different branches/apps